src/containers/ClusterTasks/ClusterTasks.js

```diff
@@ -68,52 +85,30 @@ export /* istanbul ignore next */ class ClusterTasksContainer extends Component
       );
     }
     return (
-      <StructuredListWrapper border selection>
-        <StructuredListHead>
-          <StructuredListRow head>
-            <StructuredListCell head>ClusterTask</StructuredListCell>
-            <StructuredListCell head />
-          </StructuredListRow>
-        </StructuredListHead>
-        <StructuredListBody>
-          {!clusterTasks.length && (
-            <StructuredListRow>
-              <StructuredListCell>No ClusterTasks</StructuredListCell>
-            </StructuredListRow>
+      <>
+        <h1>ClusterTasks</h1>
+        <Table
+          headers={initialHeaders}
+          rows={clusterTasksFormatted}
+          loading={loading}
+          selectedNamespace={selectedNamespace}
+          emptyText={intl.formatMessage(
```

Comment:
it should be `emptyTextAllNamespaces` & please elaborate more the default messages for both text props.

Reply:
I've updated it to be `emptyTextAllNamespaces` but I thought the message was appropriate as clusterTasks aren't specific to a single namespace. Please tell me if I'm wrong :)

Reply:
If that's the case then we can leave the message like that! Now, given that both messages are the same ill forward a comment Alan left in my PR with a similar situation: https://github.com/tektoncd/dashboard/pull/797#discussion_r353822648.